src/Command/FixContextCommand.php

```diff
@@ -18,12 +18,15 @@
 use Sonata\ClassificationBundle\Model\ContextInterface;
 use Sonata\ClassificationBundle\Model\ContextManagerInterface;
 use Sonata\ClassificationBundle\Model\TagManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+#[AsCommand(name: 'sonata:classification:fix-context', description: 'Generate the default context if none defined and attach the context to all elements')]
 final class FixContextCommand extends Command
 {
+    // TODO: Remove static properties when support for Symfony < 6.0 is dropped.
```

Comment:
I guess this can be removed when dropping support for Symfony < 5.4

Reply:
The attribute was introduced at 6.0. So it is better to remove both at 6.0.

Reply:
https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Console/Attribute/AsCommand.php Looks like it was already in 5.4

Reply:
Yes, and maybe is not correct documented, but the change where they make the AsCommand load lazy is on 6.0

Reply:
https://symfony.com/doc/5.4/console/commands_as_services.html#lazy-loading

vs

https://symfony.com/doc/current/console/commands_as_services.html#lazy-loading

Reply:
I thought it was because I read it in https://symfony.com/blog/new-in-symfony-5-3-lazy-command-description

Reply:
Seems like you are right even though it is not documented. Will try to update those indications on next pR.